src/nvim/drawline.c

```diff
@@ -472,7 +472,7 @@ static void apply_cursorline_highlight(win_T *wp, linenr_T lnum, int *line_attr,
         && qf_current_entry(wp) == lnum) {
       *line_attr = hl_combine_attr(*cul_attr, *line_attr);
     } else {
-      *line_attr = *cul_attr;
+      *line_attr = hl_combine_attr(*line_attr, *cul_attr);
```

Comment:
Hmm, I don't think this changes anything related to `QuickFixLine`. `QuickFixLine` is already combined with `CursorLine` in the other branch, but in a different order from Vim.

Reply:
Oh wait, has `QuickFixLine` even been applied here?